lib/validation/schema/specVersion/kind/project/module.json

```diff
@@ -149,6 +184,18 @@
 					"$ref": "../project.json#/definitions/builder-settings"
 				}
 			}
+		},
+		"builder-specVersion-3.1": {
+			"type": "object",
+			"additionalProperties": false,
+			"properties": {
+				"resources": {
+					"$ref": "../project.json#/definitions/builder-resources"
+				},
+				"settings": {
+					"$ref": "../project.json#/definitions/builder-settings"
+				}
```

Comment:
I think this also makes sense to add, but we never talked about it, right?

Reply:
Settings were already allowed before. See https://github.com/SAP/ui5-project/pull/639/files#diff-7b0836b8d51420484c25fc8977caba9234b6efc7193d3f0507731f291f8362beR183 and https://github.com/SAP/ui5-project/pull/639/files#diff-a21c72474996eb0b320d874991042a771b23940e75ce1ce9344261143d8bf875R184

Reply:
Thanks. I missed that.